config/annotations.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const Joi = require('joi');
+
+// List of data, eventually key-value pairs
+const SCHEMA_ANNOTATIONS = Joi.array()
```

Comment:
Hm my understanding was that all the annotations were a list of key-value pairs aka an array. But no worries - will change to an object.